views/index.ejs

```diff
@@ -28,6 +28,10 @@
     <script defer type="text/javascript" src="/js/vendor/lazy_loader.js"></script>
     <script defer type="text/javascript" src="/js/rtcApp.js"></script>
 
+    <% if (adobeTrackingUrl) { %>
```

Comment:
Looks like the updated requirements ask that we load the script with async. Should we add that in the code, as part of the adobeTracingUrl variable, or add a new variable?

Reply:
@rhainer we should add that in the script tag another variable is not needed for that:
`<script src="<%=adobeTrackingUrl%>" async></script>`
check the `async` attribute